test_runner/src/main/kotlin/ftl/adapter/environment/ListIPBlocks.kt

```diff
@@ -0,0 +1,27 @@
+package ftl.adapter.environment
+
+import com.google.testing.model.Date
+import com.google.testing.model.DeviceIpBlock
+import ftl.data.IpBlock
+import ftl.gc.deviceIPBlocks
+import ftl.reports.api.twoDigitString
+import java.util.Objects
+
+fun ipBlocksList() = deviceIPBlocks().toIpBlocks()
```

Comment:
```suggestion
fun ipBlocksList(): List<IpBlock> = deviceIPBlocks().toIpBlocks()
```

It's good to provide return type explicitly for public functions.
It makes the implementation a little bit easier to understand.